openssl-dynamic/src/main/java/io/netty/internal/tcnative/SSL.java

```diff
@@ -598,6 +598,20 @@ public static native boolean setCipherSuites(long ssl, String ciphers)
     public static native void setCertificateBio(
             long ssl, long certBio, long keyBio, String password) throws Exception;
 
+    /**
+     * Load a private key from the used OpenSSL ENGINE via the
+     * <a href="https://www.openssl.org/docs/man1.1.0/crypto/ENGINE_load_private_key.html">ENGINE_load_private_key</a>
+     * function.
+     *
+     * <p>Be sure you understand how OpenSsl will behave with respect to reference counting!
```

Comment:
This comment is ominous, I don't really know what I should do about it.  Maybe call `unloadPrivateKey` or something like that?  I don't see that method though.  What should a reader do?

Reply:
@carl-mastrangelo added some more docs :)